lib/stages/BeamInspect.cpp

```diff
@@ -0,0 +1,54 @@
+#include "BeamInspect.hpp"
+
+#include "visUtil.hpp" // for frameID, modulo
+#include "BeamMetadata.hpp"
+#include "StageFactory.hpp"    // for REGISTER_KOTEKAN_STAGE
+#include "buffer.h"
+#include "Telescope.hpp"
+
+using kotekan::bufferContainer;
+using kotekan::Config;
+using kotekan::Stage;
+
+REGISTER_KOTEKAN_STAGE(BeamInspect);
+
+STAGE_CONSTRUCTOR(BeamInspect) {
```

Comment:
Didn't know this macro existed, good to know. I'll start using it in the future.

Reply:
Yeah I wrote it years ago, but always forget to use it.